python/doc/examples/data_analysis/manage_data_and_samples/plot_quantile_estimation_wilks.py

```diff
@@ -60,49 +30,82 @@
 
 # %%
-# Empirical Quantile Estimator
 empiricalQuantile = sample.computeQuantile(alpha)
+print(empiricalQuantile)
 
-# Get the indices of the confidence interval bounds
-aAlpha = ot.Normal(1).computeQuantile((1.0 + beta) / 2.0)[0]
-min_i = int(N * alpha - aAlpha * m.sqrt(N * alpha * (1.0 - alpha)))
-max_i = int(N * alpha + aAlpha * m.sqrt(N * alpha * (1.0 - alpha)))
-# print(min_i, max_i)
+# %%
+# The asymptotic confidence interval of level :math:`\beta` is :math:`\left[ X_{(i_n)}, X_{(j_n)}\right]`
+# such that:
+#
+# .. math::
+#
+#     i_n & = \lfloor n \alpha - \sqrt{n}a_{\beta}\sqrt{\alpha(1-\alpha)}\rfloor\\
+#     j_n & = \lfloor n \alpha + \sqrt{n}a_{\beta}\sqrt{\alpha(1-\alpha)}\rfloor
+#
+# where  :math:`a_{\beta}` is the quantile of order :math:`(1-\beta/2)` of the standard normal distribution.
+#
+# Then we have:
+#
+# .. math::
+#
+#     \lim\limits_{n \rightarrow +\infty} \Prob{x_{\alpha} \in \left[ X_{(i_n,n)}, X_{(j_n,n)}\right]} = 1-\beta
+#
+
+# %%
+a_beta = ot.Normal(1).computeQuantile((1.0 + beta) / 2.0)[0]
+i_n = int(n * alpha - a_beta * m.sqrt(n * alpha * (1.0 - alpha)))
+j_n = int(n * alpha + a_beta * m.sqrt(n * alpha * (1.0 - alpha)))
+print(i_n, j_n)
 
 # Get the sorted sample
 sortedSample = sample.sort()
 
-# Get the Confidence interval of the Empirical Quantile Estimator [infQuantile, supQuantile]
-infQuantile = sortedSample[min_i - 1]
-supQuantile = sortedSample[max_i - 1]
+# Get the asymptotic confidence interval :math:`\left[ X_{(i_n)}, X_{(j_n)}\right]`
+# Care: the index in the sorted sample is :math:`i_n-1` and :math:`j_n-1`
+infQuantile = sortedSample[i_n - 1]
+supQuantile = sortedSample[j_n - 1]
 print(infQuantile, empiricalQuantile, supQuantile)
 
 # %%
-# Wilks number
-i = N - (min_i + max_i) // 2  # compute wilks with the same sample size
-wilksNumber = ot.Wilks.ComputeSampleSize(alpha, beta, i)
-print("wilksNumber =", wilksNumber)
+# The empirical quantile was estimated with the :math:`\lfloor \sampleSize\alpha \rfloor` -th order statistics evaluated on
+# the sample of size :math:`\sampleSize`.
+# We define :math:`i = \sampleSize-\lfloor \sampleSize\alpha \rfloor` and we evaluate the minimum sample size :math:`\tilde{\sampleSize}` that
+# ensures that the :math:`(\tilde{\sampleSize}-i)` order statistics is greater than :math:`x_{\alpha}` with the confidence :math:`\beta`.
+
+# %%
+i = n - int(n * alpha)
+minSampleSize = ot.Wilks.ComputeSampleSize(alpha, beta, i)
+print(minSampleSize)
+
+# %%
+# Here we directly ask for the evaluation of the upper bounded confidence interval:
+# the Wilks class estimates the previous minimum sample size, generates a
+# sample with that size and extracts the empirical quantile of order :math:`(\tilde{\sampleSize}-i)`.
 
 # %%
-# Wilks Quantile Estimator
 algo = ot.Wilks(output)
-wilksQuantile = algo.computeQuantileBound(alpha, beta, i)
-print("wilks Quantile 0.95 =", wilksQuantile)
+upperBoundQuantile = algo.computeQuantileBound(alpha, beta, i)
+print(upperBoundQuantile)
```

Comment:
In this case, this is a lower bound of the quantile, not an upper bound.
```suggestion
lowerBoundQuantile = algo.computeQuantileBound(alpha, beta, i)
print(lowerBoundQuantile)
```